package.json

```diff
@@ -248,65 +249,74 @@
+    "execa": "^9.6.1",
     "fast-glob": "^3.2.7",
-    "jest": "^30.2.0",
     "npm-run-all": "^4.1.5",
     "postcss": "^8.5.6",
+    "postcss-html": "^1.8.0",
+    "postcss-markdown": "^1.3.0",
     "postcss-sass": "^0.5.0",
     "postcss-scss": "^4.0.9",
     "postcss-styled-syntax": "^0.7.1",
-    "prettier": "^3.6.2",
-    "stylelint": "^16.26.0",
+    "prettier": "^3.7.4",
+    "stylelint": "github:stylelint/stylelint#dbf0945f41b84e0ff1d4572d438b97b89602b1a8",
     "stylelint-processor-glamorous": "^0.3.0",
     "stylelint-processor-styled-components": "^1.10.0",
-    "stylelint-scss": "^6.12.1",
-    "ts-jest": "^29.4.5",
-    "ts-node": "^10.9.2",
+    "stylelint-scss": "^6.13.0",
+    "tsx": "^4.21.0",
     "typed-emitter": "^2.1.0",
     "typescript": "^5.9.3",
-    "typescript-eslint": "^8.48.0"
+    "typescript-eslint": "^8.48.1",
+    "vitest": "^4.0.15"
   },
   "scripts": {
     "build": "tsc -p tsconfig.src.json",
     "build-bundle-watch": "run:p build-watch bundle-watch",
     "build-watch": "npm run build -- --watch",
     "bundle": "npm run bundle-base -- --sourcemap",
-    "bundle-base": "ts-node --transpile-only -P tsconfig.scripts.json scripts/bundle.ts",
+    "bundle-base": "tsx scripts/bundle.ts",
     "bundle-watch": "npm run bundle-base -- --sourcemap --watch",
     "build-bundle": "run-s build bundle",
     "format": "prettier . --write",
     "lint": "run-s \"lint:*\"",
     "lint:formatting": "prettier . --check",
-    "lint:js": "eslint --cache",
+    "lint:js": "eslint --cache --concurrency=auto",
     "lint:spelling": "cspell lint --no-progress",
     "lint:types": "tsc -b",
-    "lint:unit-tests": "ts-node --transpile-only -P tsconfig.scripts.json scripts/enforce-unit-tests-per-file.ts",
-    "test": "npm run build-bundle && npm run jest -- ",
+    "lint:unit-tests": "tsx scripts/enforce-unit-tests-per-file.ts",
+    "test": "npm run build-bundle && npm run vitest --",
     "test:e2e": "node scripts/run-e2e.js",
-    "test:integration": "npm run jest -- --projects test/integration",
-    "test:unit": "npm run jest -- --projects test/unit",
+    "test:integration": "npm run vitest -- --project integration",
+    "test:unit": "npm run vitest -- --project unit",
     "test:node-versions": "node scripts/test-node-versions.js",
-    "jest": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "vitest": "vitest run",
     "vscode:prepublish": "npm run build && npm run bundle-base -- --minify",
-    "publish": "vsce publish --no-git-tag-version --no-update-package-json"
+    "vscode:launch": "node scripts/launch-vscode.js",
+    "publish": "vsce publish --no-git-tag-version --no-update-package-json",
+    "switch-stylelint": "node scripts/switch-stylelint.mjs"
   },
   "prettier": "@stylelint/prettier-config",
   "volta": {
-    "node": "22.13.1",
+    "node": "22.17.0",
     "npm": "11.0.0"
+  },
+  "overrides": {
+    "stylelint-scss": {
+      "stylelint": "^17.0.0"
```

Comment:
Necessary because stylelint-scss is requiring a 16.x version as a peer dependency. Seems to work fine with v17, though, perhaps we need a patch in stylelint-scss to expand the supported version?